modules/network-observability-tcp-flag-syn-flood.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-tcp-flag-syn-flood_{context}"]
+= Detecting SYN flooding using the FlowMetric API and TCP flags
+You can create an `alertRule` to alert for SYN flooding. 
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetric*.
+. In the *Project:*  dropdown list, select the project of the Network Observability Operator instance. 
+. Click *Create FlowMetric*.
+. Create `FlowMetric` resources to add the following configurations: 
++
+.FlowMetric configuration to count flows per destination host and resource, with TCP flags
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flows-with-flags-per-destination
+spec:
+  metricName: flows_with_flags_per_destination_total
+  type: Counter
+  labels: [SrcSubnetLabel,DstSubnetLabel,DstK8S_Name,DstK8S_Type,DstK8S_HostName,DstK8S_Namespace,Flags]
+----
+====
++
+.FlowMetric configuration to count flows per source host and resource, with TCP flags
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flows-with-flags-per-source
+spec:
+  metricName: flows_with_flags_per_source_total
+  type: Counter
+  labels: [DstSubnetLabel,SrcSubnetLabel,SrcK8S_Name,SrcK8S_Type,SrcK8S_HostName,SrcK8S_Namespace,Flags]
+----
+====
+
+. Deploy the following `AlertingRule` to alert for SYN flooding:
```

Comment:
Is there a term that can be added after `AlertingRule` for clarity? Like resource?
```suggestion
. Deploy the following `AlertingRule` resource to alert for SYN flooding:
```

Reply:
Good call